tools/config.schema.yaml

```diff
@@ -0,0 +1,855 @@
+    "noiseavgenergy","noisesdenergy","noiseminenergy","noisemaxenergy","voicesumenergy",
+    "voiceavgenergy","voicesdenergy","voiceminenergy","voicemaxenergy","silencesensedfraction","noisesensedfraction",
+    "voicesensedfraction","unknownsensedfraction","silenceexpectedfraction","noiseexpectedfraction","voiceexpectedfraction",
+    "unknownexpectedfraction","countconversation"]
+                  RECORDING_MINUTES: 
+                    type: integer
+                  PAUSED_MINUTES:
+                    type: integer
+        additionalProperties: 
+          $ref: "#/definitions/PROVIDER"
+  
+  PHONE_DATA_YIELD:
+    type: object
+    required: [SENSORS, PROVIDERS]
+    properties:
+      SENSORS:
+        type: ["null", array]
+        items:
+          type: string
+      PROVIDERS:
+        type: ["null", object]
+        properties:
+          RAPIDS:
+            allOf:
+              - $ref: "#/definitions/PROVIDER"
+              - properties:
+                  FEATURES:
+                    uniqueItems: True
+                    items:
+                      type: string
+                      enum: [ratiovalidyieldedminutes, ratiovalidyieldedhours]
+                  MINUTE_RATIO_THRESHOLD_FOR_VALID_YIELDED_HOURS:
+                    type: number
+                    minimum: 0
+                    maximum: 1
+        additionalProperties: 
+          $ref: "#/definitions/PROVIDER"
+                  
+  PHONE_KEYBOARD:
+    type: object
+    required: [TABLE, PROVIDERS]
+    properties:
+      TABLE: 
+        type: string
+      PROVIDERS:
+        type: ["null", object]
+        additionalProperties: 
+          $ref: "#/definitions/PROVIDER"
+
+  PHONE_LIGHT:
+    type: object
+    required: [TABLE, PROVIDERS]
+    properties:
+      TABLE:
+        type: string
+      PROVIDERS:
+        type: ["null", object]
+        properties:
+          RAPIDS:
+            allOf:
+              - $ref: "#/definitions/PROVIDER"
+              - properties:
+                  FEATURES:
+                    uniqueItems: True
+                    items:
+                      type: string
+                      enum: ["count", "maxlux", "minlux", "avglux", "medianlux", "stdlux"]
+        additionalProperties: 
+          $ref: "#/definitions/PROVIDER"
+
+  PHONE_LOCATIONS:
+    type: object
+    required: [TABLE, LOCATIONS_TO_USE, FUSED_RESAMPLED_CONSECUTIVE_THRESHOLD, PROVIDERS]
+    properties:
+      TABLE: 
+        type: string
+      LOCATIONS_TO_USE:
+        type: string
+        enum: [ALL, GPS, ALL_RESAMPLED, FUSED_RESAMPLED]
+      FUSED_RESAMPLED_CONSECUTIVE_THRESHOLD:
+        type: integer
+        exclusiveMinimum: 0
+      FUSED_RESAMPLED_TIME_SINCE_VALID_LOCATION:
+        type: integer
+        exclusiveMinimum: 0
+      PROVIDERS:
+        type: ["null", object]
+        properties:
+          DORYAB:
+            allOf:
+                - $ref: "#/definitions/PROVIDER"
+                - properties:
```

Comment:
the rest of the config parameters are missing like DBSCAN_EPS

Reply:
fixed